src/times.js

```diff
@@ -21,9 +21,13 @@ var _curry2 = require('./internal/_curry2');
  *      R.times(R.identity, 5); //=> [0, 1, 2, 3, 4]
  */
 module.exports = _curry2(function times(fn, n) {
-  var len = Number(n);
-  var list = new Array(len);
-  var idx = 0;
+  var len = Number(n),
+      idx = 0,
+      list;
```

Comment:
I've also slowly been converted to 1:1.  But the more important thing is to keep consistent.  Definitely not a deal-breaker.